xwayland/server.c

```diff
@@ -332,15 +345,24 @@ static bool server_start(struct wlr_xwayland_server *server) {
 		pid_t pid = fork();
 		if (pid < 0) {
 			wlr_log_errno(WLR_ERROR, "second fork failed");
+			(void) !write(p[1], "", 0);
```

Comment:
Can we drop the `!`?

Reply:
Without the `!`, gcc still complains that the result of `write` is unused.